ci_config.json

```diff
@@ -105,6 +105,21 @@
       "glib:tests=false"
     ]
   },
+  "harfbuzz": {
+    "_comment": ["- Currently builds with no warnings other than an incorrect warning() function",
+                 "  tracked at https://github.com/harfbuzz/harfbuzz/issues/3378",
+                 "  Absolutely impossible to fix on Windows."],
+    "fatal_warnings": false,
+    "build_options": [
+      "glib:tests=false",
+      "cairo:tests=disabled",
+      "pixman:tests=disabled"
```

Comment:
Since commit 6f7eac16c18d8fbf088ded8015d3e6da314cdd22 it should not be necessary to manually suppress tests for other subprojects anymore.